web/modules/custom/joinup_rdf/joinup_rdf.install

```diff
@@ -21,34 +22,16 @@ function joinup_rdf_requirements(string $phase): array {
   $requirements = [];
 
   // Ensure that leftover graphs from the ADMS validation graph do not exist.
-  $requirements['joinup_rdf_adms_validation_graphs'] = [
-    'title' => t('RDF: ADMS graphs leftovers'),
-    'description' => t('Detects if there are leftover graphs and data from the ADMS validation step of the import pipeline. Can occur if the user navigates away during the validation.'),
+  $requirements['joinup_rdf_content_integrity'] = [
+    'title' => t('RDF content integrity'),
+    'description' => t('A series of requirements that ensure the consistency in the SPARQL database.'),
     'severity' => REQUIREMENT_OK,
+    'value' => t('No issues found.'),
   ];
 
-  $query = <<<QUERY
-SELECT DISTINCT ?g WHERE { GRAPH ?g {?s ?p ?o} } ORDER BY ?g
-QUERY;
-
-  $connection = \Drupal::getContainer()->get('sparql.endpoint');
-  $graphs = $connection->query($query);
-  $errors = [];
-  foreach ($graphs as $graph) {
-    $uri = $graph->g->getUri();
-    if (strpos($uri, 'http://adms-validator/') === 0) {
-      // @todo: Would it be better to just clear them? Then that would run at
-      // every deployment.
-      $errors[] = $uri;
-    }
-  }
-
-  if (!empty($errors)) {
-    $error = t('Leftover graphs were found: :graphs', [
-      ':graphs' => implode(', ', $errors),
-    ]);
-    $requirements['joinup_rdf_adms_validation_graphs']['severity'] = REQUIREMENT_ERROR;
-    $requirements['joinup_rdf_adms_validation_graphs']['value'] = $error;
+  if ($errors = \Drupal::getContainer()->get('joinup_rdf.requirements_helper')->getRequirementErrors()) {
+    $requirements['joinup_rdf_content_integrity']['severity'] = REQUIREMENT_ERROR;
+    $requirements['joinup_rdf_content_integrity']['value'] = implode("\n", $errors);
```

Comment:
The output of this concatenation looks very bad.

<img width="1451" alt="Status_report___Joinup" src="https://user-images.githubusercontent.com/473868/96413123-d8fb3900-11f3-11eb-805f-be48e3673076.png">

It looks like it was never manually tested in the UI 🙃 Why not using real HTML markup? Basic markup, such as `<ul>` and `<li>`, is something that even `drush core:requirements` knows how to handle .

Just setting the `value` to a render array with `'#theme' => 'item_list'` and removing the hardcoded dash should do it.

Reply:
They look much better now :) 

![Selection_137](https://user-images.githubusercontent.com/12171851/96430730-3484f100-120b-11eb-9b55-a74dd7238f0c.png)
![Selection_138](https://user-images.githubusercontent.com/12171851/96430739-38b10e80-120b-11eb-8c6f-4098b6205f0d.png)